Libraries/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ add_subdirectory(LibAudio)
 add_subdirectory(LibC)
 add_subdirectory(LibCore)
 add_subdirectory(LibCrypto)
+add_subdirectory(LibCrypt)
```

Comment:
"LibC" comes before "LibCore". This is also the usual way of how dictionaries are sorted: prefix first, then the longer words. If we want to follow this principle, "LibCrypt" should come before "LibCrypto".